src/server/game/Battlegrounds/Battleground.h

```diff
@@ -137,6 +137,8 @@ enum BattlegroundTimeIntervals
     BUFF_RESPAWN_TIME               = 180,                  // secs
 };
 
+#define RESURRECTION_INTERVAL (sWorld->getIntConfig(CONFIG_BATTLEGROUND_RESPAWN_TIME) * IN_MILLISECONDS)
```

Comment:
No use defining it if it is only used in one place imo.
Isn't it just better to call the config value like in BattgroundMgr.cpp?

Reply:
RESURRECTION_INTERVAL is used in many other places, and calling the config value was throwing a Codacy error.

Reply:
ah true, didnt think of it. Only where you edited :D

But then you can use that defined value in BattlegroundMgr too, instead of calling it "manually"?

Reply:
Was thinking that, didn't know if it would work.

Reply:
still WIP ?

Reply:
Polish is done, should be ready to test/merge.